ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -7872,3 +7877,56 @@ module VTPM = struct
     let ref = Client.VTPM.get_by_uuid ~rpc ~session_id ~uuid in
     Client.VTPM.destroy ~rpc ~session_id ~self:ref
 end
+
+module Tracing = struct
+  let create _printer rpc session_id params =
+    let name_label = List.assoc "name-label" params in
+    let hosts =
+      List.assoc_opt "hosts" params
+      |> Option.fold ~none:[] ~some:(fun hosts ->
+             List.map
+               (fun uuid -> Client.Host.get_by_uuid ~rpc ~session_id ~uuid)
+               (String.split_on_char ';' hosts)
```

Comment:
Use commas as separator, other uuid parameters use that (lines 1349, 1379, 1421, etc)